WordPress/WordPressTest/FeatureFlagTests.swift

```diff
@@ -1,11 +1,29 @@
 import XCTest
 @testable import WordPress
 
-fileprivate enum MockFeatureFlag: OverrideableFlag {
+enum MockFeatureFlag: OverrideableFlag {
```

Comment:
I believe that if you make this conform to `CaseIterable` you should be able to get all cases by using `MockFeatureFlag.allCases` (and not require the `remoteCases` array below).

Ref: https://www.simpleswiftguide.com/how-to-iterate-over-enum-cases-in-swift/

Reply:
Yeah! I don't actually want all the cases – just the remote ones (so not `enabledFeature`, `disabledFeature`, and `nonOverrideableFeature` – I thought about using `CaseIterable` and filtering out the ones I _didn't_ want but thought maybe that'd be too clever? 🤷‍♂️

This way is a bit more explicit, if verbose 😅

Reply:
It's fine I didn't realize it was some.